test/e2e/logs_multi_pipeline_broken_pipeline_test.go

```diff
@@ -0,0 +1,78 @@
+//go:build e2e
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/loggen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+)
+
+var _ = Describe(suite.ID(), Label(suite.LabelLogs), Ordered, func() {
+	Context("When a broken logpipeline exists", Ordered, func() {
+		var (
+			mockNs              = suite.IDWithSuffix("broken-pipeline")
```

Comment:
No need for a suffix since you split the files
```suggestion
			mockNs              = suite.ID()
```

Reply:
Please do it in all the other files.